docs/v2/guides/context.md

```diff
@@ -150,10 +150,22 @@ $context['post'] = Timber::get_post()->setup();
 
 ### Archive templates
 
-The `posts` variable will be available in archive templates (when [ `is_archive()`](https://developer.wordpress.org/reference/functions/is_archive/) returns `true`), like your posts index page, category or tag archives, date-based or author archives. It will contain a `Timber\PostCollection` object with the posts that WordPress already fetched for your archive page.
+The `posts` variable will be available in archive templates (when [ `is_archive()`](https://developer.wordpress.org/reference/functions/is_archive/) returns `true`). In addition to that, it will also contain `post` or `term` variables for different archive types. Here’s a small overview.
+
+| Archive | Condition | Context variables |
+|---|---|---|
+| Home | `is_home()` | `post`<br>`posts` |
+| Taxonomy Archive | `is_category()`<br>`is_tag()`<br>`is_tax()` | `term`<br>`posts` |
+| Author Archive | `is_author()` | `author`<br>`posts` |
+| Search Archive | `is_search()` | `posts`<br>`search_query` |
+| All other archives | `is_archive()` | `posts` |
```

Comment:
This is awesome. I wouldn't have thought to tabulate this info but this will be super useful!